inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py

```diff
@@ -0,0 +1,272 @@
+    matmul_2_kernel = ngraph.constant(
+        weights[weights_offset:weights_offset + matmul_2_kernel_length].reshape(matmul_2_kernel_shape)
+    )
+    weights_offset += matmul_2_kernel_length
+    matmul_2_node = ngraph.matmul(reshape_2_node, matmul_2_kernel, False, True)
+
+    # add 4
+    add_4_kernel_shape, add_4_kernel_length = shape_and_length([1, 10])
+    add_4_kernel = ngraph.constant(
+        weights[weights_offset:weights_offset + add_4_kernel_length].reshape(add_4_kernel_shape)
+    )
+    weights_offset += add_4_kernel_length
+    add_4_node = ngraph.add(matmul_2_node, add_4_kernel)
+
+    # softmax
+    softmax_axis = 1
+    softmax_node = ngraph.softmax(add_4_node, softmax_axis)
+
+    # result
+    result_node = ngraph.result(softmax_node)
+
+    # nGraph function
+    function = Function(result_node, [param_node], 'lenet')
+
+    return function
+
+
+def main():
+    log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.INFO, stream=sys.stdout)
+    args = build_argparser().parse_args()
+
+    input_images = list_input_images(args.input)
+
+    # Loading network using ngraph function
+    ngraph_function = create_ngraph_function(args)
+    net = IENetwork(Function.to_capsule(ngraph_function))
+
+    assert len(net.input_info.keys()) == 1, "Sample supports only single input topologies"
+    assert len(net.outputs) == 1, "Sample supports only single output topologies"
+
+    log.info("Preparing input blobs")
+    input_blob = next(iter(net.input_info))
+    out_blob = next(iter(net.outputs))
+    net.batch_size = len(input_images)
+
+    # Read and pre-process input images
+    n, c, h, w = net.input_info[input_blob].input_data.shape
+    images = np.ndarray(shape=(n, c, h, w))
+    for i in range(n):
+        image = read_image(input_images[i])
+        assert image is not None, log.error("Can't open an image {}".format(input_images[i]))
+        assert len(image.shape) == 2, log.error('Sample supports images with 1 channel only')
+        if image.shape[:] != (w, h):
+            log.warning("Image {} is resized from {} to {}".format(input_images[i], image.shape[:], (w, h)))
+            image = cv2.resize(image, (w, h))
+        images[i] = image
+    log.info("Batch size is {}".format(n))
+
+    log.info("Creating Inference Engine")
+    ie = IECore()
+
+    log.info('Loading model to the device')
+    exec_net = ie.load_network(network=net, device_name=args.device.upper())
+
+    # Start sync inference
+    log.info('Creating infer request and starting inference')
+    res = exec_net.infer(inputs={input_blob: images})
+
+    # Processing results
+    log.info("Processing output blob")
+    res = res[out_blob]
+    log.info("Top {} results: ".format(args.number_top))
+    if args.labels:
+        with open(args.labels, 'r') as f:
+            labels_map = [x.split(sep=' ', maxsplit=1)[-1].strip() for x in f]
+    else:
+        labels_map = None
```

Comment:
you can remove else and set labels_map = None before if statement

Reply:
not an issue, but looks better, fixed